### PR TITLE
[INLONG-10135][Manager] Move inlongCompressType to clustertag configuration

### LINKED ISSUE #10135: [Improve][Manager] Move inlongCompressType to clustertag configuration
### Description

Move inlongCompressType to clustertag configuration.

### InLong Component

InLong Manager

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by fuweng11**
### Prepare a Pull Request
- Fixes #10135 

### Motivation

inlongCompressType to clustertag configuration
### Modifications

inlongCompressType to clustertag configuration

### CODE REVIEW

File: inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/cluster/ClusterTagRequest.java [hunk 1]
```diff
@@ -55,6 +55,9 @@ public class ClusterTagRequest {
     @Length(min = 1, max = 163840, message = "length must be between 1 and 163840")
     private String extParams;
 
+    @ApiModelProperty(value = "The compression type used for dataproxy and sort side data transmission to reduce the network IO overhead")
+    private String inlongCompressType;
```
Comment (aloyszhang) [inline inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/cluster/ClusterTagRequest.java:59]:
```suggestion
    private String compressType;
```

File: inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/cluster/ClusterTagResponse.java [hunk 2]
```diff
@@ -50,6 +50,9 @@ public class ClusterTagResponse {
     @ApiModelProperty(value = "Extended params")
     private String extParams;
 
+    @ApiModelProperty(value = "The compression type used for dataproxy and sort side data transmission to reduce the network IO overhead")
+    private String inlongCompressType;
```
Comment (aloyszhang) [inline inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/cluster/ClusterTagResponse.java:54]:
```suggestion
    private String ompressType;
```

File: inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/cluster/InlongClusterTagExtParam.java [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@ApiModel("Inlong cluster tag ext param info")
+public class InlongClusterTagExtParam implements Serializable {
+
+    private static final Gson GSON = new Gson();
+
+    @ApiModelProperty(value = "The compression type used for dataproxy and sort side data transmission to reduce the network IO overhead")
+    private String inlongCompressType = "NONE";
```
Comment (aloyszhang) [inline inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/cluster/InlongClusterTagExtParam.java:52]:
```suggestion
    private String compressType = "NONE";
```

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/repository/DataProxyConfigRepository.java [hunk 4]
```diff
@@ -99,6 +100,7 @@ public class DataProxyConfigRepository implements IRepository {
     public static final String KEY_DATA_NODE_NAME = "defaultDataNodeName";
     public static final String KEY_SORT_CONSUMER_GROUP = "defaultSortConsumerGroup";
     public static final String KEY_SINK_NAME = "defaultSinkName";
+    public static final String KEY_INLONG_COMPRESS_TYPE = "inlongCompressType";
```
Comment (aloyszhang) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/repository/DataProxyConfigRepository.java:103]:
```suggestion
    public static final String KEY_COMPRESS_TYPE = "compressType";
```

File: inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/cluster/ClusterTagResponse.java (hunk 2 above)
Comment (fuweng11) [reply inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/cluster/ClusterTagResponse.java:54]: This name is used to distinguish it from other compress types, such as hive's compress type.

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/repository/DataProxyConfigRepository.java (hunk 4 above)
Comment (fuweng11) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/repository/DataProxyConfigRepository.java:103]: This name is used to distinguish it from other compress types, such as hive's compress type.

File: inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/cluster/InlongClusterTagExtParam.java (hunk 3 above)
Comment (fuweng11) [reply inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/cluster/InlongClusterTagExtParam.java:52]: This name is used to distinguish it from other compress types, such as hive's compress type, and this variable name is used in other components.

File: inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/cluster/ClusterTagRequest.java (hunk 1 above)
Comment (fuweng11) [reply inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/cluster/ClusterTagRequest.java:59]: This name is used to distinguish it from other compress types, such as hive's compress type, and this variable name is used in other components

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/repository/DataProxyConfigRepository.java (hunk 4 above)
Comment (aloyszhang) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/repository/DataProxyConfigRepository.java:103]: All right.

**Review by aloyszhang — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

**Review by gosonzhang — APPROVED** [review]

**Review by luchunliang — APPROVED** [review]

**Pull request merged**